ocs-plugin-spi/src/main/java/de/cxp/ocs/elasticsearch/query/model/QueryFilterTerm.java

```diff
@@ -0,0 +1,41 @@
+package de.cxp.ocs.elasticsearch.query.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.apache.lucene.search.BooleanClause.Occur;
+
+@Data
+@AllArgsConstructor
+@RequiredArgsConstructor
+@NoArgsConstructor
+public class QueryFilterTerm implements QueryStringTerm {
+
+    @NonNull
+    private String	word;
+    @NonNull
+    private String  field;
+
+    private Occur   occur			= Occur.MUST;
+
+    public QueryFilterTerm(String field, String word, boolean isOrFilter) {
+        this.field = field;
+        this.word = word;
+    }
+
+    @Override
+    public String toQueryString() {
+
+        return occur.toString()
+            + field
+            + ":"
+            + word;
```

Comment:
done (StringEscapeUtils.javaescape())

Reply:
Sorry, I was not precise enough. I was talking about escaping Elasticsearch specific characters, like quotes, pluses etc. There is EscapeUtil.escapeReservedESCharacters for that...

Reply:
I mean.. actually it is not required here at the moment, since we use the value as term-filters anyways. :thinking:  
However in case someone gives us a rule like `title:foo*` we should consider to escape the wildcard to avoid it being executed as wildcard.. or.. should we allow it? I am not sure... maybe we should.. ok, let's decide later.

But with this thoughts it just came to my mind, that a filter like `title:foo` would not work at the moment, because a title field most likely is not filterable... we should keep that in mind as well or leave a TODO comment..